src/core/FileUpload/FileUploadTemplate.tsx

```diff
@@ -42,25 +47,26 @@ export type FileUploadTemplateProps = {
 export const FileUploadTemplate = (props: FileUploadTemplateProps) => {
   const {
     onChange,
-    acceptMultiple = true,
+    acceptMultiple,
     label = 'Choose a file',
     subtitle = 'or drag & drop it here.',
     children,
+    inputProps,
   } = props;
   useTheme();
 
   return (
     <>
-      <input
-        className='iui-browse-input'
-        type='file'
-        id='file-input'
-        onChange={onChange}
-        multiple={acceptMultiple}
-      />
       <SvgUpload className='iui-icon' aria-hidden />
       <div className='iui-template-text'>
-        <label htmlFor='file-input' className='iui-anchor'>
```

Comment:
hmm... Wasnt there a reason that input was used separately from label and connected with htmlFor?
Somehow I feel there was something, that we needed to do it only that way...

Reply:
well ok, seems like its still working, so.. i guess its ok...